### PR TITLE
remove manual formatting

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/cli/doc.py [hunk 1]
```diff
@@ -1124,22 +1111,21 @@ def add_fields(text, fields, limit, opt_indent, return_values=False, base_indent
...
-                if isinstance(opt[k], string_types):
-                    text.append('%s%s: %s' % (opt_indent, k,
-                                              textwrap.fill(DocCLI.tty_ify(opt[k]),
-                                                            limit - (len(k) + 2),
-                                                            subsequent_indent=opt_indent)))
-                elif isinstance(opt[k], (Sequence)) and all(isinstance(x, string_types) for x in opt[k]):
-                    text.append(DocCLI.tty_ify('%s%s: %s' % (opt_indent, k, ', '.join(opt[k]))))
+
+                if isinstance(opt[k], Sequence) and not isinstance(opt[k], string_types):
+                    text.append(DocCLI._dump_yaml({k: opt[k]}, opt_indent, flow_style=True))
```
Comment (felixfontein) [inline lib/ansible/cli/doc.py]: For some reason this causes
```
        {aliases: [attr]}
        {choices: [all, 'no', none, safe, urllib2, 'yes']}
        {default: [200]}
```
while non-sequence values for `default` yield
```
        default: 30
```
without the `{...}`.

**Review by felixfontein — COMMENTED** [review]
Right now there are several newlines inserted at strange places:
```
        - test_certificates
            Determines which certificates in the chain will be tested.
            `all' tests all certificates in the chain (excluding the leaf, which is identical in all chains).
            `first' only tests the first certificate in the chain, that is the one which signed the leaf.
            `last' only tests the last certificate in the chain, that is the one furthest away from the leaf. Its issuer is the root certificate of this
            chain.
            {choices: [first, last, all]}
            
            default: all
            
            type: str
            
```
Maybe `yaml_dump` needs an `rstrip('\n')`?

**Comment by bcoca** [comment]
we insert those at some points, need to rearrange that, also rewrite many of the tests since this does change the expected output, I was just showing the core of the changes needed, it is more about deleting code than adding.

File: lib/ansible/cli/doc.py (hunk 1 above)
Comment (bcoca) [reply lib/ansible/cli/doc.py]: just need to yaml_dump the value and use 'stringification' direclty on key

**Comment by samccann** [comment]
@bcoca are you planning on including this before the 2.14 freeze date?

**Comment by bcoca** [comment]
I don't' have the time to work on this right now, feel free to run with it, if not, i'll eventually get to it.

**Comment by felixfontein** [comment]
I started creating a fork of the branch here: #78668

**Pull request closed without merging**